spec/requests/api/v0/properties/search_request_spec.rb

```diff
@@ -36,7 +36,7 @@
       expect(property[:attributes]).to be_a(Hash)
 
       attributes = property[:attributes]
-      keys = [:street, :city, :state, :zip, :walk_score, :transit_score, :bike_score, :safety_score]
+      keys = [:street, :city, :state, :zip]
```

Comment:
I've been meaning to ask you about how you test like that! I remembered Mike talking about it during our first solo project review and then saw you do it on something. I wanted to do it on the last project but never remembered how you did it.